fmp_server.py

```diff
@@ -40,6 +40,237 @@ def recvAnsConnection(self, packet_id, data, seq):
         self.server.debug("Connection: {!r}".format(connection_data))
         self.sendNtcLogin(3, connection_data, seq)
 
+    def recvReqLayerJump(self, packet_id, data, seq):
+        """ReqLayerJump packet.
+        
+        ID: 64100100
+        JP: レイヤ移動要求（位置バイナリ）
+        TR: Layer move request (position binary)
+        
+        A relay of the target user's layer_host.
```

Comment:
I was thinking about that, and my idea was that they would have a method of communicating to the other server to reserve a slot. That way you avoid having an issue of state changing mid flight. But I don't if they actually did that or if it would be fast enough to not create a noticeable slowdown in the process.